tests/Aspire.Templates.Tests/WorkloadTestsBase.cs

```diff
@@ -4,6 +4,7 @@
 using System.Text;
```

Comment:
Not necessarily in this PR, but can we rename this file? It still says "WorkloadTestBase", but the class is TemplateTestBase

Reply:
Oops, good catch, thanks!